x/slashing/client/cli/query.go

```diff
@@ -0,0 +1,57 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/tendermint/tmlibs/cli"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/wire" // XXX fix
+	"github.com/cosmos/cosmos-sdk/x/slashing"
+)
+
+// get the command to query signing info
+func GetCmdQuerySigningInfo(storeName string, cdc *wire.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "signing_info [owner-addr]",
```

Comment:
pubkey or owner-address? when this is referenced it adds it to a pubkey

Reply:
Updated, thanks